sdk/eventhubs/azure-messaging-eventhubs/migration-guide.md

```diff
@@ -1,142 +1,148 @@
+client libraries have.
 
-- The latest dependency for `azure-messaging-eventhubs` is available [here](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/eventhubs/azure-messaging-eventhubs#including-the-package).
-- The latest dependency for `azure-messaging-eventhubs-checkpointstore-blob` is available [here](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob#including-the-package).
+There were several areas of consistent feedback expressed across the Azure client library ecosystem. One of the most important is that the client libraries for different Azure services have not had a consistent organization, naming, and API structure. Additionally, many developers have felt that the learning curve was difficult, and the APIs did not offer a good, approachable, and consistent onboarding story for those learning Azure or exploring a specific Azure service.
 
-## General changes
+To improve the development experience across Azure services, a set of uniform [design guidelines][Guidelines] was created for all languages to drive a
+consistent experience with established API patterns for all services. A set of [Java design guidelines][GuidelinesJava] was introduced to ensure that Java clients have a natural and idiomatic feel with respect to the Java ecosystem. Further details are available in the guidelines
+for those interested.
 
-Version 5 of the Azure Event Hubs SDK is a result of our efforts to create a client library that is user-friendly and idiomatic to the Java ecosystem.
+### Cross Service SDK improvements
 
-Apart from redesigns resulting from the new [Azure SDK Design Guidelines for Java](https://azure.github.io/azure-sdk/java_introduction.html), the latest version improves on several areas from V3.
+The modern Event Hubs client library also provides the ability to share in some of the cross-service improvements made to the Azure development experience, such as
 
-In the interest of simplifying the API surface, we've made three clients, each with an asynchronous and synchronous
-variant. One client is for producing events, `EventHubProducerAsyncClient`, while two are intended for reading events.
-`EventProcessorClient` is the production-level consumer and `EventHubConsumerAsyncClient` for exploration and
-lower-level control of `EventData` consumption.
+- Using the new azure-identity library to share a single authentication approach between clients.
+- A unified logging and diagnostics pipeline offering a common view of the activities across each of the client libraries.
+- A unified asynchronous programming model using [Project Reactor][project-reactor].
 
```

Comment:
If this apply, you may want to add this
`- Use of builders to build the client.`